packages/core/theme-module/index.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path')
+const fs = require("fs")
+const consola = require('consola')
+const chalk = require('chalk');
+const chokidar = require('chokidar')
+
+const log = {
+  info: (message) => consola.info(chalk.bold('VSF'), message),
+  success: (message) => consola.success(chalk.bold('VSF'), message),
+  warning: (message) => consola.warning(chalk.bold('VSF'), message),
+  error: (message) => consola.error(chalk.bold('VSF'), message)
+}
+
+const getAllFiles = function(dirPath) {
+  files = fs.readdirSync(dirPath)
+ 
+  arrayOfFiles = []
+ 
+  files.forEach(function(file) {
```

Comment:
Just use map:

```JS
files.map(file => {
   if (fs.statSync(`${dirPath}/${file}`).isDirectory()) {
     return ...getAllFiles(dirPath + '/' + file, arrayOfFiles)
   } else {
      return`${dirPath}/${file}`.split(__dirname + '/').pop()
})
```

Reply:
I would say reduce...  but... it's just a proposition:
```
return files.reduce((prev, curr) => {
  if (fs.statSync(dirPath + "/" + curr).isDirectory()) {
    return getAllFiles(dirPath + "/" + curr, prev)
  }

  const newFiles = (dirPath + "/" + curr).split(__dirname + '/').pop()

  return [...prev, ...newFiles]
}, [])
```